src/command-line/users/index.js

```diff
@@ -1,6 +1,9 @@
 "use strict";
 
-if (!require("../../helper").config.ldap.enable) {
+if (
+	!require("../../helper").config.ldap.enable &&
+	!require("../../helper").config.headerAuth.enable
```

Comment:
if you use it multiple times, import it once at the toplevel please, don't repeat require like this.